scripts/build-json/slice-prose.js

```diff
@@ -49,14 +53,18 @@ function packageValues(heading, dom) {
 
 function getSection(node, sections) {
     const sectionName = node.textContent.trim();
-    const sectionContent = extractFromSiblings(node, '#comment', 'html');
+    const sectionContent = extractFromSiblings(node, 'H2', 'html');
     const extraSections = [];
 
     if (namedSections.includes(sectionName)) {
-        sections[sectionName] = sectionContent;
+        const sectionSlug = sectionNameToSlug(sectionName);
+        sections[sectionSlug] = {
+            title: sectionName,
+            content: sectionContent
+        }
```

Comment:
I'm trying to comprehend how we now deal with someone editing `prose.md` and type in `Short Description`. Had we just stuck to `short_description` (or `short-description` depending on the angle) we'd be safe. However, there's definitely intrinsic value in consistency and a breakage here would alert a contributor that we prefer "Short description" instead of "Short Description" and if you don't adhere it will break the rendering.